### PR TITLE
fall-protection: prevent spam

### PR DESCRIPTION
**Pull request by RedstoneFuture**
Currently, the player get a message with ever seconds about the fall-protection time, after death. This may be too much, so I suggest, to send the message only with the start of fall-protection. The second message is still the info about the end of it. This is enough, I think.

### CODE REVIEW

**Comment by daniel-naegele** [comment]
If you want to change this, please make it configurable. On some servers, the fall protection is used for starting to ride on the missile and the countdown is helpful for this. We should give server owners the choice at which seconds the player gets notified.

**Comment by RedstoneFuture** [comment]
I added a config option to support booth behaviors about sending the fall protection message: only on first time or periodic.

File: missilewars-plugin/src/main/java/de/butzlabben/missilewars/wrapper/abstracts/arena/FallProtectionConfiguration.java [hunk 1]
```diff
@@ -30,4 +30,5 @@
 
     private final boolean enabled = true;
     private final int duration = 60;
+    private final boolean message_only_on_start = false;
```
Comment (daniel-naegele) [inline missilewars-plugin/src/main/java/de/butzlabben/missilewars/wrapper/abstracts/arena/FallProtectionConfiguration.java]: Please use the Java convention for variable naming (camelCase). You can modify the serialized name with the annotation
@SerializedName("message_only_on_start")

**Review by daniel-naegele — CHANGES_REQUESTED** [review]

File: missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java [hunk 2]
```diff
@@ -246,7 +246,7 @@ public void onRespawn(PlayerRespawnEvent e) {
             e.setRespawnLocation(t.getSpawn());
             FallProtectionConfiguration fallProtection = getGame().getArena().getFallProtection();
             if (fallProtection.isEnabled())
-                new RespawnGoldBlock(e.getPlayer(), fallProtection.getDuration(), getGame());
+                new RespawnGoldBlock(e.getPlayer(), fallProtection.getDuration(), fallProtection.isMessage_only_on_start(), getGame());
```
Comment (daniel-naegele) [inline missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java]: This call has to be renamed, as the automatic getter generated by lombok ist now named isMessageOnlyOnStart(). With the current version, the project won't even compile

**Review by daniel-naegele — CHANGES_REQUESTED** [review]

File: missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java (hunk 2 above)
Comment (RedstoneFuture) [reply missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java]: -.- Sorry.

**Review by daniel-naegele — APPROVED** [review]

**Pull request merged**